core/src/main/java/org/mobilitydata/gtfsvalidator/notice/PointNearOriginNotice.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * A point is too close to origin (0, 0).
```

Comment:
Reading the code in the subsequent files I suggest the following change to the docs:

```suggestion
 * A point is less than @code{1.0} (unit to be detailed) from the origin.
```

Reply:
I am not sure that this is necessary. I do not see why that is important for reader. There is an ocean in the rectangle from from (10,10) to (-10,10) - absolutely no land there. The typical bug is to export coordinates (0,0). We use a tiny threshold to avoid problems with rounding.

By the way, you probably wanted to write `{@code 1.0}` instead of `@code{1.0}`. Also, there is no "unit" - 1 is just a degree.

Reply:
> I am not sure that this is necessary. I do not see why that is important for reader. There is an ocean in the rectangle from from (10,10) to (-10,10) - absolutely no land there. The typical bug is to export coordinates (0,0). We use a tiny threshold to avoid problems with rounding.

Thanks for the explanation. 

> By the way, you probably wanted to write {@code 1.0} instead of @code{1.0}. 

Indeed, thanks for flagging that.

> Also, there is no "unit" - 1 is just a degree.

Makes sense to me now, thanks @aababilov!